packages/shared/src/components/post/customPrompts/PromptButtons.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { forwardRef, useState } from 'react';
+import type { ReactElement, Ref } from 'react';
+import { ColorName } from '../../../styles/colors';
+import {
+  ArrowIcon,
+  CustomPromptIcon,
+  EditPromptIcon,
+  TLDRIcon,
+} from '../../icons';
+import type { ButtonProps } from '../../buttons/Button';
+import {
+  Button,
+  ButtonIconPosition,
+  ButtonSize,
+  ButtonVariant,
+} from '../../buttons/Button';
+import { IconSize } from '../../Icon';
+import { usePromptsQuery } from '../../../hooks/prompt/usePromptsQuery';
+import { ElementPlaceholder } from '../../ElementPlaceholder';
+import type { PromptFlags } from '../../../graphql/prompt';
+import { PromptDisplay } from '../../../graphql/prompt';
+import { usePromptButtons } from '../../../hooks/feed/usePromptButtons';
+import { useViewSize, ViewSize } from '../../../hooks';
+import { SimpleTooltip } from '../../tooltips';
+
+export const PromptIconMap = {
+  TLDR: TLDRIcon,
+  CustomPrompt: CustomPromptIcon,
+  EditPrompt: EditPromptIcon,
+};
+
+type PromptButtonProps = ButtonProps<'button'> & {
+  active: boolean;
+  flags: PromptFlags;
+};
+
+const PromptButton = forwardRef(
+  (
+    { children, flags, active, ...props }: PromptButtonProps,
+    ref?: Ref<HTMLButtonElement>,
+  ): ReactElement => {
+    const PromptIcon = PromptIconMap[flags.icon] || CustomPromptIcon;
+    const variant = active ? ButtonVariant.Primary : ButtonVariant.Subtle;
+    const color = active ? flags.color : undefined;
+    return (
+      <Button
+        variant={variant}
+        color={color}
+        size={ButtonSize.XSmall}
+        icon={
+          <PromptIcon
+            size={IconSize.XSmall}
+            className={!active && `text-accent-${flags.color}-default`}
```

Comment:
This will break tailwind, we should probably actually do a manual map function that fully qualifies or store full classnames?

Reply:
https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Reply:
Fixed in 0b3fe5a